### ISSUE TITLE
Interceptor - unable to inject @Resource

### ISSUE THREAD

**Issue by zerovcs** [payara/Payara#2314]
# Description #
----------

When using interceptor binding `@Resource` injection throws a runtime exception. 

Annotation:
```
@InterceptorBinding
@Target({ElementType.TYPE, ElementType.METHOD}) 
@Retention(RetentionPolicy.RUNTIME)
@Inherited
public @interface Monitored {
}
```
Interceptor:
```
@Monitored
@Interceptor
@Priority(Interceptor.Priority.APPLICATION)
public class BindingInterceptor implements Serializable
{
	@Resource SessionContext ses;
	@AroundInvoke
	public Object intercept(InvocationContext context) throws Exception{
		return context.proceed();
	}
}
```
SessionBean:
```
@Stateless
@LocalBean
@ExcludeDefaultInterceptors
@Monitored
public class SB2 {
	@PostConstruct
	public void init() {
		System.out.println(">>> SB2 @PostConstruct ..." + this); 
	}
	public void ping() {
		System.out.println(">>> SB2 ping ..." + this);
	}
}
```

EJB method ping is invoked from Servlet:
```
@WebServlet("/servlet1")
public class MyServlet1 extends HttpServlet {
	private static final long serialVersionUID = 1L;
       
	@EJB SB2 sb2;
	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
		sb2.ping();
		response.getWriter().append("Served at: ").append(request.getContextPath()).append(" ").append(new Date().toString());
	}
}
```
When ping is invoked an exception is thrown:

> 2018-01-23T13:53:18.574+0100|Severe: ejb.stateless_ejbcreate_exception
> 2018-01-23T13:53:18.575+0100|Warning: A system exception occurred during an invocation on EJB SB2, method: public void si.test.SB2.ping()
> 2018-01-23T13:53:18.575+0100|Warning: javax.ejb.EJBException: javax.ejb.EJBException: javax.ejb.CreateException: Could not create stateless EJB
> 	at com.sun.ejb.containers.StatelessSessionContainer._getContext(StatelessSessionContainer.java:448)
> 	at com.sun.ejb.containers.BaseContainer.getContext(BaseContainer.java:2596)
> 	at com.sun.ejb.containers.BaseContainer.preInvoke(BaseContainer.java:1987)
> 	at com.sun.ejb.containers.EJBLocalObjectInvocationHandler.invoke(EJBLocalObjectInvocationHandler.java:210)
> 	at com.sun.ejb.containers.EJBLocalObjectInvocationHandlerDelegate.invoke(EJBLocalObjectInvocationHandlerDelegate.java:90)
> 	at com.sun.proxy.$Proxy296.ping(Unknown Source)
> 	at si.test.__EJB31_Generated__SB2__Intf____Bean__.ping(Unknown Source)
> 	at si.test.MyServlet1.doGet(MyServlet1.java:34)
> 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:687)
> 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
> 	at org.apache.catalina.core.StandardWrapper.service(StandardWrapper.java:1692)
> 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:258)
> 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:160)
> 	at org.apache.catalina.core.StandardPipeline.doInvoke(StandardPipeline.java:654)
> 	at org.apache.catalina.core.StandardPipeline.invoke(StandardPipeline.java:593)
> 	at com.sun.enterprise.web.WebPipeline.invoke(WebPipeline.java:99)
> 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:159)
> 	at org.apache.catalina.connector.CoyoteAdapter.doService(CoyoteAdapter.java:371)
> 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:238)
> 	at com.sun.enterprise.v3.services.impl.ContainerMapper$HttpHandlerCallable.call(ContainerMapper.java:483)
> 	at com.sun.enterprise.v3.services.impl.ContainerMapper.service(ContainerMapper.java:180)
> 	at org.glassfish.grizzly.http.server.HttpHandler.runService(HttpHandler.java:206)
> 	at org.glassfish.grizzly.http.server.HttpHandler.doHandle(HttpHandler.java:180)
> 	at org.glassfish.grizzly.http.server.HttpServerFilter.handleRead(HttpServerFilter.java:235)
> 	at org.glassfish.grizzly.filterchain.ExecutorResolver$9.execute(ExecutorResolver.java:119)
> 	at org.glassfish.grizzly.filterchain.DefaultFilterChain.executeFilter(DefaultFilterChain.java:284)
> 	at org.glassfish.grizzly.filterchain.DefaultFilterChain.executeChainPart(DefaultFilterChain.java:201)
> 	at org.glassfish.grizzly.filterchain.DefaultFilterChain.execute(DefaultFilterChain.java:133)
> 	at org.glassfish.grizzly.filterchain.DefaultFilterChain.process(DefaultFilterChain.java:112)
> 	at org.glassfish.grizzly.ProcessorExecutor.execute(ProcessorExecutor.java:77)
> 	at org.glassfish.grizzly.nio.transport.TCPNIOTransport.fireIOEvent(TCPNIOTransport.java:539)
> 	at org.glassfish.grizzly.strategies.AbstractIOStrategy.fireIOEvent(AbstractIOStrategy.java:112)
> 	at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.run0(WorkerThreadIOStrategy.java:117)
> 	at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.access$100(WorkerThreadIOStrategy.java:56)
> 	at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy$WorkerThreadRunnable.run(WorkerThreadIOStrategy.java:137)
> 	at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.doWork(AbstractThreadPool.java:593)
> 	at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.run(AbstractThreadPool.java:573)
> 	at java.lang.Thread.run(Thread.java:745)
> Caused by: javax.ejb.EJBException: javax.ejb.CreateException: Could not create stateless EJB
> 	at com.sun.ejb.containers.StatelessSessionContainer$SessionContextFactory.create(StatelessSessionContainer.java:713)
> 	at com.sun.ejb.containers.util.pool.NonBlockingPool.getObject(NonBlockingPool.java:220)
> 	at com.sun.ejb.containers.StatelessSessionContainer._getContext(StatelessSessionContainer.java:444)
> 	... 37 more
> Caused by: javax.ejb.CreateException: Could not create stateless EJB
> 	at com.sun.ejb.containers.StatelessSessionContainer.createStatelessEJB(StatelessSessionContainer.java:527)
> 	at com.sun.ejb.containers.StatelessSessionContainer.access$000(StatelessSessionContainer.java:101)
> 	at com.sun.ejb.containers.StatelessSessionContainer$SessionContextFactory.create(StatelessSessionContainer.java:711)
> 	... 39 more
> Caused by: java.lang.IllegalStateException: Exception attempting to inject Env-Prop: si.test.interceptor.BindingInterceptor/ses@Field-Injectable Resource. Class name = si.test.interceptor.BindingInterceptor Field name=ses@java.lang.String@si.test.interceptor.BindingInterceptor/ses@@ into class si.test.interceptor.BindingInterceptor: Lookup failed for 'java:comp/env/si.test.interceptor.BindingInterceptor/ses' in SerialContext[myEnv={java.naming.factory.initial=com.sun.enterprise.naming.impl.SerialInitContextFactory, java.naming.factory.state=com.sun.corba.ee.impl.presentation.rmi.JNDIStateFactoryImpl, java.naming.factory.url.pkgs=com.sun.enterprise.naming}
> 	at org.glassfish.weld.services.InjectionServicesImpl.aroundInject(InjectionServicesImpl.java:185)
> 	at org.jboss.weld.injection.InjectionContextImpl.run(InjectionContextImpl.java:46)
> 	at org.jboss.weld.injection.producer.ResourceInjector.inject(ResourceInjector.java:72)
> 	at org.jboss.weld.injection.producer.BasicInjectionTarget.inject(BasicInjectionTarget.java:117)
> 	at org.jboss.weld.bean.ManagedBean.create(ManagedBean.java:159)
> 	at org.jboss.weld.context.unbound.DependentContextImpl.get(DependentContextImpl.java:70)
> 	at org.jboss.weld.bean.ContextualInstanceStrategy$DefaultContextualInstanceStrategy.get(ContextualInstanceStrategy.java:100)
> 	at org.jboss.weld.bean.ContextualInstance.get(ContextualInstance.java:50)
> 	at org.jboss.weld.manager.BeanManagerImpl.getReference(BeanManagerImpl.java:785)
> 	at org.jboss.weld.manager.BeanManagerImpl.getReference(BeanManagerImpl.java:808)
> 	at org.glassfish.weld.services.JCDIServiceImpl.createInterceptorInstance(JCDIServiceImpl.java:338)
> 	at com.sun.ejb.containers.BaseContainer.injectEjbInstance(BaseContainer.java:1774)
> 	at com.sun.ejb.containers.StatelessSessionContainer.createStatelessEJB(StatelessSessionContainer.java:488)
> 	... 41 more
> Caused by: com.sun.enterprise.container.common.spi.util.InjectionException: Exception attempting to inject Env-Prop: si.test.interceptor.BindingInterceptor/ses@Field-Injectable Resource. Class name = si.test.interceptor.BindingInterceptor Field name=ses@java.lang.String@si.test.interceptor.BindingInterceptor/ses@@ into class si.test.interceptor.BindingInterceptor: Lookup failed for 'java:comp/env/si.test.interceptor.BindingInterceptor/ses' in SerialContext[myEnv={java.naming.factory.initial=com.sun.enterprise.naming.impl.SerialInitContextFactory, java.naming.factory.state=com.sun.corba.ee.impl.presentation.rmi.JNDIStateFactoryImpl, java.naming.factory.url.pkgs=com.sun.enterprise.naming}
> 	at com.sun.enterprise.container.common.impl.util.InjectionManagerImpl._inject(InjectionManagerImpl.java:740)
> 	at com.sun.enterprise.container.common.impl.util.InjectionManagerImpl.inject(InjectionManagerImpl.java:507)
> 	at org.glassfish.weld.services.InjectionServicesImpl.aroundInject(InjectionServicesImpl.java:129)
> 	... 53 more
> Caused by: javax.naming.NamingException: Lookup failed for 'java:comp/env/si.test.interceptor.BindingInterceptor/ses' in SerialContext[myEnv={java.naming.factory.initial=com.sun.enterprise.naming.impl.SerialInitContextFactory, java.naming.factory.state=com.sun.corba.ee.impl.presentation.rmi.JNDIStateFactoryImpl, java.naming.factory.url.pkgs=com.sun.enterprise.naming} [Root exception is javax.naming.NameNotFoundException: No object bound to name java:comp/env/si.test.interceptor.BindingInterceptor/ses]
> 	at com.sun.enterprise.naming.impl.SerialContext.lookup(SerialContext.java:496)
> 	at com.sun.enterprise.naming.impl.SerialContext.lookup(SerialContext.java:442)
> 	at javax.naming.InitialContext.lookup(InitialContext.java:417)
> 	at com.sun.enterprise.container.common.impl.util.InjectionManagerImpl._inject(InjectionManagerImpl.java:638)
> 	... 55 more
> Caused by: javax.naming.NameNotFoundException: No object bound to name java:comp/env/si.test.interceptor.BindingInterceptor/ses
> 	at com.sun.enterprise.naming.impl.GlassfishNamingManagerImpl.lookup(GlassfishNamingManagerImpl.java:761)
> 	at com.sun.enterprise.naming.impl.GlassfishNamingManagerImpl.lookup(GlassfishNamingManagerImpl.java:735)
> 	at com.sun.enterprise.naming.impl.JavaURLContext.lookup(JavaURLContext.java:159)
> 	at com.sun.enterprise.naming.impl.SerialContext.lookup(SerialContext.java:476)
> 	... 58 more

if instead of SessionContext trying to inject DataSource;
```
        private DataSource ds;
	@Resource(lookup="jdbc/__default")   
	public void setDs(DataSource dataSource) {
		this.ds = dataSource;
	}

```

> 2018-01-23T14:12:38.576+0100|Severe: ejb.stateless_ejbcreate_exception
> 2018-01-23T14:12:38.576+0100|Warning: A system exception occurred during an invocation on EJB SB2, method: public void si.test.SB2.ping()
> 2018-01-23T14:12:38.577+0100|Warning: javax.ejb.EJBException: javax.ejb.EJBException: javax.ejb.CreateException: Could not create stateless EJB
> 	at com.sun.ejb.containers.StatelessSessionContainer._getContext(StatelessSessionContainer.java:448)
> 	at com.sun.ejb.containers.BaseContainer.getContext(BaseContainer.java:2596)
> 	at com.sun.ejb.containers.BaseContainer.preInvoke(BaseContainer.java:1987)
> 	at com.sun.ejb.containers.EJBLocalObjectInvocationHandler.invoke(EJBLocalObjectInvocationHandler.java:210)
> 	at com.sun.ejb.containers.EJBLocalObjectInvocationHandlerDelegate.invoke(EJBLocalObjectInvocationHandlerDelegate.java:90)
> 	at com.sun.proxy.$Proxy301.ping(Unknown Source)
> 	at si.test.__EJB31_Generated__SB2__Intf____Bean__.ping(Unknown Source)
> 	at si.test.MyServlet1.doGet(MyServlet1.java:34)
> 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:687)
> 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
> 	at org.apache.catalina.core.StandardWrapper.service(StandardWrapper.java:1692)
> 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:258)
> 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:160)
> 	at org.apache.catalina.core.StandardPipeline.doInvoke(StandardPipeline.java:654)
> 	at org.apache.catalina.core.StandardPipeline.invoke(StandardPipeline.java:593)
> 	at com.sun.enterprise.web.WebPipeline.invoke(WebPipeline.java:99)
> 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:159)
> 	at org.apache.catalina.connector.CoyoteAdapter.doService(CoyoteAdapter.java:371)
> 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:238)
> 	at com.sun.enterprise.v3.services.impl.ContainerMapper$HttpHandlerCallable.call(ContainerMapper.java:483)
> 	at com.sun.enterprise.v3.services.impl.ContainerMapper.service(ContainerMapper.java:180)
> 	at org.glassfish.grizzly.http.server.HttpHandler.runService(HttpHandler.java:206)
> 	at org.glassfish.grizzly.http.server.HttpHandler.doHandle(HttpHandler.java:180)
> 	at org.glassfish.grizzly.http.server.HttpServerFilter.handleRead(HttpServerFilter.java:235)
> 	at org.glassfish.grizzly.filterchain.ExecutorResolver$9.execute(ExecutorResolver.java:119)
> 	at org.glassfish.grizzly.filterchain.DefaultFilterChain.executeFilter(DefaultFilterChain.java:284)
> 	at org.glassfish.grizzly.filterchain.DefaultFilterChain.executeChainPart(DefaultFilterChain.java:201)
> 	at org.glassfish.grizzly.filterchain.DefaultFilterChain.execute(DefaultFilterChain.java:133)
> 	at org.glassfish.grizzly.filterchain.DefaultFilterChain.process(DefaultFilterChain.java:112)
> 	at org.glassfish.grizzly.ProcessorExecutor.execute(ProcessorExecutor.java:77)
> 	at org.glassfish.grizzly.nio.transport.TCPNIOTransport.fireIOEvent(TCPNIOTransport.java:539)
> 	at org.glassfish.grizzly.strategies.AbstractIOStrategy.fireIOEvent(AbstractIOStrategy.java:112)
> 	at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.run0(WorkerThreadIOStrategy.java:117)
> 	at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.access$100(WorkerThreadIOStrategy.java:56)
> 	at org.glassfish.grizzly.strategies.WorkerThreadIOStrategy$WorkerThreadRunnable.run(WorkerThreadIOStrategy.java:137)
> 	at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.doWork(AbstractThreadPool.java:593)
> 	at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.run(AbstractThreadPool.java:573)
> 	at java.lang.Thread.run(Thread.java:745)
> Caused by: javax.ejb.EJBException: javax.ejb.CreateException: Could not create stateless EJB
> 	at com.sun.ejb.containers.StatelessSessionContainer$SessionContextFactory.create(StatelessSessionContainer.java:713)
> 	at com.sun.ejb.containers.util.pool.NonBlockingPool.getObject(NonBlockingPool.java:220)
> 	at com.sun.ejb.containers.StatelessSessionContainer._getContext(StatelessSessionContainer.java:444)
> 	... 37 more
> Caused by: javax.ejb.CreateException: Could not create stateless EJB
> 	at com.sun.ejb.containers.StatelessSessionContainer.createStatelessEJB(StatelessSessionContainer.java:527)
> 	at com.sun.ejb.containers.StatelessSessionContainer.access$000(StatelessSessionContainer.java:101)
> 	at com.sun.ejb.containers.StatelessSessionContainer$SessionContextFactory.create(StatelessSessionContainer.java:711)
> 	... 39 more
> Caused by: java.lang.IllegalStateException: Exception attempting to inject Res-Ref-Env-Property: si.test.interceptor.BindingInterceptor/ds@javax.sql.DataSource@ resolved as: jndi: jdbc/__default@res principal: null@mail: null
> No Runtime properties
> Database Vendor : null
> Create Tables at Deploy : false
> Delete Tables at Undeploy : false into class si.test.interceptor.BindingInterceptor: Lookup failed for 'java:comp/env/si.test.interceptor.BindingInterceptor/ds' in SerialContext[myEnv={java.naming.factory.initial=com.sun.enterprise.naming.impl.SerialInitContextFactory, java.naming.factory.state=com.sun.corba.ee.impl.presentation.rmi.JNDIStateFactoryImpl, java.naming.factory.url.pkgs=com.sun.enterprise.naming}
> 	at org.glassfish.weld.services.InjectionServicesImpl.aroundInject(InjectionServicesImpl.java:185)
> 	at org.jboss.weld.injection.InjectionContextImpl.run(InjectionContextImpl.java:46)
> 	at org.jboss.weld.injection.producer.ResourceInjector.inject(ResourceInjector.java:72)
> 	at org.jboss.weld.injection.producer.BasicInjectionTarget.inject(BasicInjectionTarget.java:117)
> 	at org.jboss.weld.bean.ManagedBean.create(ManagedBean.java:159)
> 	at org.jboss.weld.context.unbound.DependentContextImpl.get(DependentContextImpl.java:70)
> 	at org.jboss.weld.bean.ContextualInstanceStrategy$DefaultContextualInstanceStrategy.get(ContextualInstanceStrategy.java:100)
> 	at org.jboss.weld.bean.ContextualInstance.get(ContextualInstance.java:50)
> 	at org.jboss.weld.manager.BeanManagerImpl.getReference(BeanManagerImpl.java:785)
> 	at org.jboss.weld.manager.BeanManagerImpl.getReference(BeanManagerImpl.java:808)
> 	at org.glassfish.weld.services.JCDIServiceImpl.createInterceptorInstance(JCDIServiceImpl.java:338)
> 	at com.sun.ejb.containers.BaseContainer.injectEjbInstance(BaseContainer.java:1774)
> 	at com.sun.ejb.containers.StatelessSessionContainer.createStatelessEJB(StatelessSessionContainer.java:488)
> 	... 41 more
> Caused by: com.sun.enterprise.container.common.spi.util.InjectionException: Exception attempting to inject Res-Ref-Env-Property: si.test.interceptor.BindingInterceptor/ds@javax.sql.DataSource@ resolved as: jndi: jdbc/__default@res principal: null@mail: null
> No Runtime properties
> Database Vendor : null
> Create Tables at Deploy : false
> Delete Tables at Undeploy : false into class si.test.interceptor.BindingInterceptor: Lookup failed for 'java:comp/env/si.test.interceptor.BindingInterceptor/ds' in SerialContext[myEnv={java.naming.factory.initial=com.sun.enterprise.naming.impl.SerialInitContextFactory, java.naming.factory.state=com.sun.corba.ee.impl.presentation.rmi.JNDIStateFactoryImpl, java.naming.factory.url.pkgs=com.sun.enterprise.naming}
> 	at com.sun.enterprise.container.common.impl.util.InjectionManagerImpl._inject(InjectionManagerImpl.java:740)
> 	at com.sun.enterprise.container.common.impl.util.InjectionManagerImpl.inject(InjectionManagerImpl.java:507)
> 	at org.glassfish.weld.services.InjectionServicesImpl.aroundInject(InjectionServicesImpl.java:129)
> 	... 53 more
> Caused by: javax.naming.NamingException: Lookup failed for 'java:comp/env/si.test.interceptor.BindingInterceptor/ds' in SerialContext[myEnv={java.naming.factory.initial=com.sun.enterprise.naming.impl.SerialInitContextFactory, java.naming.factory.state=com.sun.corba.ee.impl.presentation.rmi.JNDIStateFactoryImpl, java.naming.factory.url.pkgs=com.sun.enterprise.naming} [Root exception is javax.naming.NameNotFoundException: No object bound to name java:comp/env/si.test.interceptor.BindingInterceptor/ds]
> 	at com.sun.enterprise.naming.impl.SerialContext.lookup(SerialContext.java:496)
> 	at com.sun.enterprise.naming.impl.SerialContext.lookup(SerialContext.java:442)
> 	at javax.naming.InitialContext.lookup(InitialContext.java:417)
> 	at com.sun.enterprise.container.common.impl.util.InjectionManagerImpl._inject(InjectionManagerImpl.java:638)
> 	... 55 more
> Caused by: javax.naming.NameNotFoundException: No object bound to name java:comp/env/si.test.interceptor.BindingInterceptor/ds
> 	at com.sun.enterprise.naming.impl.GlassfishNamingManagerImpl.lookup(GlassfishNamingManagerImpl.java:761)
> 	at com.sun.enterprise.naming.impl.GlassfishNamingManagerImpl.lookup(GlassfishNamingManagerImpl.java:735)
> 	at com.sun.enterprise.naming.impl.JavaURLContext.lookup(JavaURLContext.java:159)
> 	at com.sun.enterprise.naming.impl.SerialContext.lookup(SerialContext.java:476)
> 	... 58 more
> 

----------
Even more weird situation occurs if binding is removed (from interceptor and session bean) and explicit Interceptors annotation is used instead:
```
//@Monitored
@Interceptor
@Priority(Interceptor.Priority.APPLICATION)
public class BindingInterceptor implements Serializable {
	private DataSource ds;
	@Resource(lookup="jdbc/__default")   
	public void setDs(DataSource dataSource) {
		System.out.println(">>>>>> BindingInterceptor DI " + this + "  " + dataSource);       
		this.ds = dataSource;
	}
	@AroundInvoke
	public Object intercept(InvocationContext context) throws Exception	{
		return context.proceed();
	}
}
```

```
@Stateless
@LocalBean
@ExcludeDefaultInterceptors
//@Monitored
@Interceptors(BindingInterceptor.class)
public class SB2 {
...
}
```

No exceptions but injection is sometimes performed, sometimes not. I couldn't found out a pattern, but it is enough to republish application to change behaviour (using Eclipse as IDE)

But if I remove all annotations and define a default interceptor (in ejb-jar.xml) then injection works fine al the time

## Environment ##

- **Payara Version**: 4.1.2.174
- **Edition**: Full
- **JDK Version**: 8 u51 - Oracle
- **Operating System**: Windows
- **Database**:  Derby

**Comment by mulderbaba** [comment]
probably related to PAYARA-2229.

**Comment by rdebusscher** [comment]
Hi @zerovcs ,

The fix for PAYARA-2229 is available in versions 4.1.2.182 and 5.182. Can you verify with these versions if the issue still occurs.

If so, can you create a small reproducer which demonstrate the issue.

Thanks
Regards
Rudy.

**Comment by OndroMih** [comment]
Hi, can you verify that the issue is resolved in the latest version or provide us a reproducer if it's not resolved? A reproducer should ideally follow the SSCCE rules: http://www.sscce.org/. If we don't hear from you soon, we'll assume that all is OK and we'll close this issue.
